src/utils/data.js

```diff
@@ -14,6 +20,12 @@ export let filterDataByUserAllocation = (datasets, allocations) => {
     return filteredDatasets
 }
 
+export let filterDataByOwner = (datasets, ownerAddress) => {
+    let filteredDatasets = []
+    filteredDatasets = datasets.filter((d) => d.owner === ownerAddress)
```

Comment:
Found the issue. Basically, our filter is not case sensitive, so it leaves out any result that doesn't match exactly. While our `d.owner` is always lowercase, `ownerAddress` is not. 

I'll add a quick fix: `datasets.filter((d) => d.owner === ownerAddress.toLowerCase())`, it should be enough.

Reply:
I see, good catch. Thanks for fixing this!